pipeline-steps/templates/jobs/fortify-scan-job.yaml

```diff
@@ -10,13 +10,15 @@ parameters:
   applicationName: ''
   rootFolder: '.'
   qualityGateLevel: 1 # 0 for no quality gate, 1 includes critical issues only, 2 includes high, 3 includes medium and 4 includes low.
-  technologyStack: 'JAVA/J2EE/Kotlin'
-  technologyVersion: '11'
+  technologyStack: 'JS/TS/HTML'
```

Comment:
This tech stack is more generic than JAVA or Python which will throw errors without the correct requirements installed, so is more applicable across the majority of our repos.